### ISSUE TITLE
MA0182 should handle COM objects

### ISSUE THREAD

**Issue by meziantou** [meziantou/Meziantou.Analyzer#958]
````c#
[ComImport]
[Guid(IIDGuid.IFileSaveDialog)]
[CoClass(typeof(FileSaveDialogRCW))]
[SuppressMessage("Style", "IDE1006:Naming Styles", Justification = "Used as a class")]
internal interface NativeFileSaveDialog : IFileSaveDialog // This class should not be reported because of CoClass. There is a usage through FileSaveDialogRCW
{
}


[ComImport]
[ClassInterface(ClassInterfaceType.None)]
[Guid(CLSIDGuid.FileSaveDialog)]
internal sealed class FileSaveDialogRCW
{
}

````
